core-libs/setup/ssr/error-handling/server-error-response-transformers/server-error-response-transformers.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { InjectionToken } from '@angular/core';
+import { Applicable } from '@spartacus/core';
+import { CxServerError } from '../server-errors';
+
+/**
+ * A transformer responsible for transforming an error into a {@link CxServerError}.
+ * The transformer is applicable when the error matches the transformer's criteria.
+ * Spartacus provides two default transformers:
+ * - {@link CmsPageNotFoundServerErrorResponseTransformer}
+ * - {@link UnknownServerErrorResponseTransformer}
```

Comment:
This part of docs is easy to become outdated. Do you think we should really put it in the JSDocs of the generic interface?